src/index.js

```diff
@@ -134,38 +136,58 @@ export const validateWhitelistOption = (whitelistOpt = []) => {
 
 let hasBeenLogged = false;
 
+const logPlugin = (plugin, targets, list) => {
+  const envList = list[plugin];
+  const filteredList = Object.keys(targets)
+  .reduce((a, b) => {
+    a[b] = envList[b];
+    return a;
+  }, {});
+  const logStr = `\n ${plugin} ${JSON.stringify(filteredList)}`;
+  console.log(logStr);
+};
+
 export default function buildPreset(context, opts = {}) {
   const loose = validateLooseOption(opts.loose);
   const moduleType = validateModulesOption(opts.modules);
   const whitelist = validateWhitelistOption(opts.whitelist);
   const targets = getTargets(opts.targets);
   const debug = opts.debug;
+  const useBuiltIns = opts.useBuiltIns;
```

Comment:
@kaicataldo asked about this as well.

It is an implementation + it's just the name we used before that's all.

https://github.com/babel/babel/pull/3327 and we went with `useBuiltIns`